packages/js/data/src/product-tags/index.ts

```diff
@@ -1,23 +1,26 @@
+/**
+ * External dependencies
+ */
+import { register } from '@wordpress/data';
+
 /**
  * Internal dependencies
  */
 import { STORE_NAME, WC_PRODUCT_TAGS_NAMESPACE } from './constants';
 import { createCrudDataStore } from '../crud';
-import { PromiseifySelectors } from '../types/promiseify-selectors';
-import { ProductTagSelectors } from './types';
+import { ProductTag, ProductTagActions, ProductTagSelectors } from './types';
 
-createCrudDataStore( {
+export const store = createCrudDataStore<
+	ProductTag,
+	ProductTagActions,
+	ProductTagSelectors
+>( {
 	storeName: STORE_NAME,
 	resourceName: 'ProductTag',
 	pluralResourceName: 'ProductTags',
 	namespace: WC_PRODUCT_TAGS_NAMESPACE,
 } );
 
-export const EXPERIMENTAL_PRODUCT_TAGS_STORE_NAME = STORE_NAME;
+register( store );
```

Comment:
I think this `register` call may be causing this: `Store "wc/admin/products/tags" is already registered.`, given `createCrudDataStore` already registers the store doesn't it?
<img width="800" alt="Screenshot 2025-02-18 at 10 52 09 AM" src="https://github.com/user-attachments/assets/e4c433f4-d92c-4b55-811c-4551039427d5" />

Reply:
Nice catch! Fixed with 788e64373b85d1f5845e329dc66e8212d24395fb!